src/mcts/search.h

```diff
@@ -300,6 +300,7 @@ class SearchWorker {
   void FetchSingleNodeResult(NodeToProcess* node_to_process,
                              int idx_in_computation);
   void DoBackupUpdateSingleNode(const NodeToProcess& node_to_process);
+  void MaybeConvertParent(Node* p, bool& can_convert, float m) const;
```

Comment:
Non-const reference parameters are not allowed by the style guide (see also https://herbsutter.com/2020/02/23/references-simply/). For out-parameters use pointer instead (and make it the last parameter).
That way it's clear at the calling site that the parameter can be changed.

Reply:
I'll just switch this to a `bool` return value